client/src/helpers/test/page-defs.spec.js

```diff
@@ -0,0 +1,100 @@
+import { PP, CV1, CV2, PC, ER, TP } from '../../inside/defs-grid/index'
+
+import { inputTypes, dependentOn } from '../ehr-types'
+import EhrDefs from '../ehr-defs-grid'
+import ehrValidations from '../../inside/components/page/ehr-validations'
+const should = require('should')
+
+const pageDefs = Object.assign({},
+  PP(),
+  CV1(),
+  CV2(),
+  PC(),
+  ER(),
+  TP()
+)
```

Comment:
Instead of this could we export the pageDefs and add comment that the export is intended for unit testing?  Or we could extract the creation of pageDefs into a standalone module and import in both places.  Please choose between these two.

Reply:
I think that the second one is best and that it is included in the suggestion for the changes for the `pageDefs` index file... so basically, we'll directly be importing pageDefs instead of combining the objects in every import. Thank you for the feedback!